cloud_sql/postgres_instance_custom_subject_alternative_names/main.tf

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+# [START cloud_sql_instance_service_identity]
+resource "google_project_service_identity" "default" {
+  provider = google-beta
+  service  = "sqladmin.googleapis.com"
+}
+# [END cloud_sql_instance_service_identity]
+
+# [START privateca_ca_pool_suffix]
```

Comment:
Same as @msampathkumar mentioned, stick consistently to the same product prefix (cloud_sql)